diesel/src/query_builder/sqlite.rs

```diff
@@ -0,0 +1,44 @@
+use backend::{Sqlite, SqliteType};
+use super::{QueryBuilder, BuildQueryResult, Context};
+use types::HasSqlType;
+
+#[doc(hidden)]
+pub struct SqliteQueryBuilder {
+    pub sql: String,
+    pub bind_params: Vec<(SqliteType, Option<Vec<u8>>)>,
+}
+
+impl SqliteQueryBuilder {
+    pub fn new() -> Self {
+        SqliteQueryBuilder {
+            sql: String::new(),
+            bind_params: Vec::new(),
+        }
+    }
+}
+
+impl QueryBuilder<Sqlite> for SqliteQueryBuilder {
+    fn push_sql(&mut self, sql: &str) {
+        self.sql.push_str(sql);
+    }
+
+    fn push_identifier(&mut self, identifier: &str) -> BuildQueryResult {
+        self.push_sql("`");
+        self.push_sql(&identifier.replace("`", "``"));
```

Comment:
I'm not sure this is required. This function is already `SqliteQueryBuilder#escape_identifier`. I'm not sure what other information to convey other than that we're escaping identifiers for SQLite, which the method/class name already give.